.github/workflows/test.yml

```diff
@@ -57,26 +47,7 @@ jobs:
             && sudo apt-get -y install podman dbus-x11 \
             && podman version
       - run: |
-          curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
-          chmod +x kind
-          sudo mv kind /bin/
-      - run: |
-          kind create cluster
-          kind export kubeconfig
-          sudo chown -R "$USER" "$HOME/.kube"
-          sudo usermod -aG docker "$USER"
```

Comment:
Was this a mistake? I didn't see anything in the scripts/start_registry bash script, but I could be looking at an out-of-date diff?

Reply:
I've done a lot of force pushing in this branch. But the GitHub PR viewer shows me the script and I see it in Visual Studio Git Graph, as well on my local system, so I think the file is still present.

Reply:
Apologies, my comment wasn't super clear. I meant removing the process of curling the kind binary down and creating a kind cluster appears to be removed in that diff.

Reply:
And looking at that script you introduced, it looks like we still want to curl/create that kind cluster, but want to replace the job that's responsible for creating and wiring up the certificates needed to properly authenticate to the local docker container registry, with that script.

Reply:
Actually the kind cluster is running directly within the test binary now, so no need to pull it down separately. See https://github.com/operator-framework/operator-registry/pull/605/files#diff-ace2659936556766681fb6406432bca037081e3c00cf76b735ff153865b167a1R70